content/_data/changelogs/weekly.yml

```diff
@@ -2352,6 +2352,35 @@
     # pull: 3309 too minor (form target fix for admin monitor)
     # pull: 3334 too minor (German translation)
     # pull: 3352 too minor (ExtensionList#lookupSingleton)
+- version: "2.113"
+  date: 2018-03-25
+  changes:
+    - type: major rfe
+      pull: 3359
+      message: >
+        Update Remoting from 3.18 to 3.19 so that Jenkins core can always deserialize exceptions even if they're not whitelisted.
+        To benefit from this improvement, Remoting needs to be updated on the agent side as well.
```

Comment:
The original message was unclear whether this is _required_ in general, but given our conservative approach to compatibility, I interpreted it to mean the update on agent side is only required to benefit from this change.

Reply:
LGTM